api/src/main/java/io/strimzi/api/kafka/model/status/KafkaConnectStatus.java

```diff
@@ -19,13 +19,15 @@
         builderPackage = "io.fabric8.kubernetes.api.builder"
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({ "conditions", "observedGeneration", "restApiAddress" })
+@JsonPropertyOrder({ "conditions", "observedGeneration", "restApiAddress", "restApiPort" })
 @EqualsAndHashCode
 public class KafkaConnectStatus extends Status {
     private static final long serialVersionUID = 1L;
 
     private String restApiAddress;
 
+    private int restApiPort;
+
     @Description("REST API endpoint for managing and monitoring Kafka Connect connectors.")
     public String getRestApiAddress() {
```

Comment:
To me `address` means `host+port`, so if what is held in this property was like `foobar:1234`, but is now `foobar` with `1234` in the `restApiAddress` then we should rename this property to `restApiHost`.

And we should use consistent language (consistent with the wording for the bridge etc) in the `@Description` to explain what this is.